app/controllers/hubstats/deploys_controller.rb

```diff
@@ -4,22 +4,22 @@ module Hubstats
   class DeploysController < ApplicationController
 
     def index
-      #deploy_id = Hubstats::Deploy
-      #  .order_with_timespan(@timespan, "ASC")
-      #  .belonging_to_users(params[:users])
-      #  .belonging_to_repos(params[:repo])
-      #  .has_many_pull_requests(params[:pull_requests])
-      #  .map(&:id)
+      deploy_id = Hubstats::Deploy
```

Comment:
I actually thought it was, but you're right, the index isn't relying on it at all.